lib/files.ncl

```diff
@@ -1,11 +1,22 @@
 let nix = import "./nix-interop/nix.ncl" in
+let RelativePath =
+  std.contract.from_predicate (fun x => !(x |> std.string.characters |> std.array.at 0 == "/"))
```

Comment:
Nitpick: I would break the line to make it easier to read.

```suggestion
  std.contract.from_predicate (fun x => !(x |> std.string.characters |> std.array.first == "/"))
```